<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoveFilesDialog</class>
 <widget class="QDialog" name="MoveFilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="userInfoTextBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>361</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectFileButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>SelectNamesFile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectPathButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>SelectPath</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveFilesButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>将Path中的文件按照Names文件中的路径信息移动</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
